GoogleService-Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I deleted and redownloaded/added this file as I was debugging. It should be the same file and its just for the Alpha build.

Reply:
Ok so let's go ahead and remove this file in this pr.